articles/migrations/guides/passwordless-start.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Migration Guide for the Use of /passwordless/start from Confidential Applications
+description: Auth0 is deprecating the usage of the /passwordless/start endpoint from confidential applications without a client secret in the request.
+topics:
+  - passwordless
+  - migrations
+contentType:
+  - concept
+  - how-to
+useCase:
+  - customize-connections
+---
+# Migration Guide: Use of /passwordless/start from Confidential Applications
+
+Auth0 is deprecating the use of the `/passwordless/start` endpoint from confidential applications when no client secret is included in the request.
+
+OAuth uses the term 'confidential' for applications that can store secrets. In Auth0, those are 'Regular Web Applications', which serve web pages from a backend app. Single Page Applications and Native Applications are considered 'public' applications, and are not affected by this change.
+
+## Does this affect me?
```

Comment:
> If your applications currently calls the `/passwordless/start` endpoint directly to begin passwordless authentication from a Web Application, this deprecation does affect you.

Since I believe we're planning to proactively migrate customers who aren't affected as of the deprecation date, this deprecation won't actually apply to customers who start making calls after that date. Could we clarify this include some dates, something like "If your applications were calling the `/passwordless/start` endpoint directly to begin passwordless authentication from a Web Application in the 60 days leading up to the deprecation date, this deprecation does affect you".  I feel like @jeffreylees might have a better way with words than I do to express that.

This also seems misleading since it doesn't mention customers could also be impacted who are using the hosted login page.

Reply:
@mmaddex you are right about not mentioning customers calling it from the custom login page. I rephrased the entire thing. I also changed the wording to avoid mentioning a date.

Please let me know if it makes sense or not cc @jeffreylees

Reply:
Makes sense to me. 👍